### DISCUSSION TITLE
Add to HACS

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [Mariusthvdb/custom-attributes#25] [Ideas]
I believe we can now publish this, and also submit to HACS. Considering the 'default' branch at HACS takes a while, would it be faster to do that otherwise?

**Comment by elchininet** [comment]
Let me create the HACS workflow and also some deployment workflow, that could manage automatic versioning and automatic releases. Then we can add the HACS Custom badge, submit it to default and wait, it could take months but meanwhile you can show it in the Home Assistant forum to spread the word.

**Reply by Mariusthvdb** [reply]
o heck, I just added that HACS validate... 
please feel free to delete that and do as you say above.

the more people can test this, the beter it will be.

before that, we might need to guard it with a issue template (requiring fields) and an FR template?

also I keep seeing this:

<img width="860" alt="Scherm­afbeelding 2023-12-13 om 22 50 58" src="https://github.com/Mariusthvdb/custom-attributes/assets/33354141/92cbffd0-1025-4bbd-a208-85c5c89482b3">

not sure what that is?

**Reply by elchininet** [reply]
Do not worry, it is a good starting point.

>before that, we might need to guard it with a issue template (requiring fields) and an FR template?

This can be done in parallel, both are not a blocker to each other.

>also I keep seeing this:

It should dissapear when the workflows are done.
